Formula/yaml.rb

```diff
@@ -0,0 +1,28 @@
+class Yaml < Formula
+  desc "Process YAML documents from the CLI"
+  homepage "https://github.com/mikefarah/yaml"
+  url "https://github.com/mikefarah/yaml/archive/1.13.1.tar.gz"
+  sha256 "5512c586c6dd4f9b838d80facde656153b4fe3c1ce4992cc3057f0930fec4c7a"
+
+  depends_on "go" => :build
+  depends_on "govendor" => :build
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/mikefarah/yaml").install contents
+
+    ENV["GOPATH"] = gopath
+    ENV.prepend_create_path "PATH", gopath/"bin"
+
+    cd gopath/"src/github.com/mikefarah/yaml" do
+      system "govendor", "sync"
+      system "go", "build", "-o", "bin/yaml"
+      bin.install "bin/yaml"
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.

Reply:
cool, done